src/Form/Form.js

```diff
@@ -18,6 +18,28 @@ function findFieldOption(options, field) {
   });
 }
 
+function mergeNewModel(stateModel, currentModelDefinition, nextModelDefinition) {
```

Comment:
i think i should rename it into `currentDefinitionModel` because `currentModel` is not what it really is. the `stateModel` is the current model